### DISCUSSION TITLE
Cannot see commits of old account in my current account's contribution graph

### DISCUSSION THREAD

**Discussion by viren-joshi** [community/community#47762] [Other Feature Feedback, Questions, & Ideas] [answered]
### Github Profile

Transfer my commit history of one account to another.

I want to basically transfer my commit history of my work email to my personal email.
Hence I followed this [GitHub Article](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts), but I still cannot see the commits I made using my work account in my current profile in the contribution graph of my personal account.

Is there a way to achieve what I want?

**Comment by viren-joshi** [comment, marked as answer]
It's working now.

**Reply by noorhashem** [reply]
Did it take a while to show up? or you did something specific? I am having the same issue.

**Reply by viren-joshi** [reply]
It took a little while to load for me, I did not do anything to fix it. So I guess yours would also show up in some time.

**Reply by noorhashem** [reply]
So I contacted the Github support and they manually rebuilt my graph, so looks like it was just a bit sluggish, now they show up.
